examples/abitest/module_1/rust/src/lib.rs

```diff
@@ -86,7 +94,13 @@ fn inner_main(in_handle: u64) -> Result<(), oak::OakStatus> {
             );
             let mut buf = Vec::<u8>::with_capacity(1024);
             let mut handles = Vec::with_capacity(1);
-            oak::channel_read(wait_handles[i], &mut buf, &mut handles)?;
+            match oak::channel_read(wait_handles[i], &mut buf, &mut handles) {
+                Ok(_) => {}
+                Err(oak::OakStatus::ERR_CHANNEL_CLOSED) => {
+                    info!("message on channel {:?} stolen elsewhere!", wait_handles[i]);
+                }
+                e => return e,
+            }
```

Comment:
More functionally (and avoiding to partly reimplement the logic behind the `?` operator):

```rust
            oak::channel_read(wait_handles[i], &mut buf, &mut handles)
              .or_else(|err| if err == oak::OakStatus::ERR_CHANNEL_CLOSED {
                info!("message on channel {:?} stolen elsewhere!", wait_handles[i]);
                // We treat ERR_CHANNEL_CLOSED as success because....
                Ok(())
              } else {
                Err(err)
              })?;
```

Reply:
Done (but I'm not entirely convinced that the result is easier to follow, particularly as I noticed that there's another case to cover).